website/manual.md

```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
 
 - [A pixelated animation one by @tanakaworld](https://github.com/denoland/deno/blob/master/website/images/deno_logo_4.gif)
 
+## Frequently Asked Questions
+
+### Is Deno production ready?
+
+No.
+
+### Will there be an official package manager for Deno?
+
+No.
+
+### How came the Deno name?
+
+Just swap the letters and `node` become `deno`.
+
+### I want to contribute, where should i start?
+
+First of all thanks for the help. You may first start to parse all the issues in
+[deno](https://github.com/denoland/deno) and
+[deno_std](https://github.com/denoland/deno) repositories. If you want to add
+functionnalities, be sure to start an issue to discuss about it. Also please
+refer to the [contributing section](#Contributing).
+
+### How Deno is different from node?
+
+- Deno is secure by default
```

Comment:
Can you expand on this? What about the language it's written in? What about resources/ops? What about libuv/tokio?  GN vs GYP? production ready vs experimental?

Reply:
Ok, i'll expand this question. It would need some review for sure.